src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ CHANGELOG
 3.4.0
 -----
 
+ * Session `use_strict_mode` is now enabled by default and the corresponding option has been deprecated,
```

Comment:
There is no consequence at all with existing sessions or any BC break?

Reply:
No technical consequences, only behaviorals, if session fixation is a feature for you (meaning you're putting your users at risk.) Can you think of any BC issue?

Reply:
@nicolas-grekas I can think of a BC: SSO!

Reply:
Did you experience it? because "session fixation" is not a feature we want to support IMHO.

Reply:
Yes, e.g. with this https://github.com/legalthings/sso - There is nothing wrong with keeping session fixation enabled for SSO, PHP's default setting for strict_mode is 0 and there are plenty of reasons for that. Could you explain why you suddenly think session fixation is unsecure after all these years?

Reply:
If you want to open that discussion, please open an issue.

Reply:
Switched to Laravel, all good now.